app/views/records/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class='categories_container'>
+  <div class='back_total'>
+    <%= link_to '< back', categories_path, class: 'back2' %>
+    <h3>$<%= @records.inject(0) {|sum,x| sum + x.amount } || 0 %></h3>
+  </div>
+  <% if @records.empty? %>
+    <h2 class='empty'> No transactions for this category.</h2>
+  <% else %>
+      <ul class='record_list'> 
+        <% @records.each do |record| %>
+           <li>
+            <p class='category_name'><%= record.name %></p>
+            <p>$<%= record.amount %></p>
+           </li>
+        <% end %>
+      </ul>  
+    </table>
+      
+  <% end %>
+  <div class='add_category_cover'>
+    <%= link_to 'Add new transaction', new_category_record_path, class: 'add_category' %>
+  </div>
+</div>
```

Comment:
- [ ] as one of the requirements is to follow the design provided you should follow colors, font-family, font-size and composition, and space between elements.
- Screenshot of the Requirements 
<img width="451" alt="Screenshot 2023-03-23 at 11 59 57" src="https://user-images.githubusercontent.com/61790587/227199009-9390b89c-7adb-4af5-9518-9a98992826e7.png">
- Font provided with colors 
<img width="437" alt="Screenshot 2023-03-23 at 12 00 24" src="https://user-images.githubusercontent.com/61790587/227200012-15900ad2-19ce-48b3-8281-329881d352f9.png">
<img width="498" alt="Screenshot 2023-03-23 at 12 00 32" src="https://user-images.githubusercontent.com/61790587/227200028-c1ab6779-e746-4065-a35d-c328d6734df8.png">


- [ ] The transactions page should be the same as the one in the design by styling the background, font , colors , the inputs 

- [ ] please make sure the button should be at the bottom of the screen and its width should be the width of the screen also as in the design 

- [ ] the background color  and spacing between elements in order to match the design 

- yours 
<img width="359" alt="Screenshot 2023-03-23 at 12 21 06" src="https://user-images.githubusercontent.com/61790587/227203102-32a4bfa8-7e41-434e-aea9-be6a092bee63.png">

- Design 
<img width="275" alt="Screenshot 2023-03-23 at 12 22 18" src="https://user-images.githubusercontent.com/61790587/227203150-070019ab-65db-4a0f-b4fb-92a755a1d3da.png">